test/axes/baseAxisTests.ts

```diff
@@ -206,4 +206,615 @@ describe("BaseAxis", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        assert.strictEqual(axis.annotationsEnabled(true), axis, "enabling annotations return calling axis");
+      });
+    });
+
+    describe("annotating ticks", () => {
+      it("has no annotated ticks by default", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        assert.deepEqual(axis.annotatedTicks(), [], "no annotated ticks by default");
+      });
+
+      it("can set the annotated ticks", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        let annotatedTicks = [20, 30, 40];
+        axis.annotatedTicks(annotatedTicks);
+        assert.deepEqual(axis.annotatedTicks(), annotatedTicks, "can set the annotated ticks");
+      });
+
+      it("returns the axis when setting the annotated ticks", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        assert.strictEqual(axis.annotatedTicks([]), axis, "setting the annotated ticks returns calling axis");
+      });
+    });
+
+    describe("formatting annotation ticks", () => {
+      it("uses the identity formatter by default", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        let annotationFormatter = axis.annotationFormatter();
+        let identityFormatter = Plottable.Formatters.identity();
+        let test = "testString";
+        assert.strictEqual(annotationFormatter(test), identityFormatter(test), "uses the identity formatter by default");
+      });
+
+      it("can set the annotation formatter", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        let annotationFormatter = (d: any) => "";
+        axis.annotationFormatter(annotationFormatter);
+        let test = "testString";
+        assert.strictEqual(annotationFormatter(test), axis.annotationFormatter()(test), "can set the annotated ticks");
+      });
+
+      it("returns the axis when setting the annotation formatter", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        assert.strictEqual(axis.annotationFormatter(() => ""), axis, "setting the annotation formatter returns calling axis");
+      });
+    });
+
+    describe("rendering annotations", () => {
+      describe("label content", () => {
+        it("does not render annotations by default", () => {
+          let mockScale = new Mocks.Scale<{}, number>();
+          let axis = new Plottable.Axis(mockScale, "bottom");
+          let svg = TestMethods.generateSVG(300, 300);
+          axis.renderTo(svg);
+          assert.strictEqual(axis.content().selectAll(".annotation-label").size(), 0, "no annotations by default");
+          svg.remove();
+        });
+
+        it("does not have annotations to render by default", () => {
```

Comment:
The previous test just amounts to saying that since annotations are disabled by default, then no annotations should render since they are disabled.

This test amounts to saying that since there are no annotated ticks; even if annotations are enabled, no annotations should render.

I do agree that the wording is a bit peculiar; how do you think this should be changed.